### PR TITLE
ngc_wrapper: Pairs file for external loopback

### PR DESCRIPTION
**Pull request by RavidRothfeld**

### CODE REVIEW

File: ngc_rdma_wrapper.sh [hunk 1]
```diff
@@ -214,19 +224,12 @@ ngc_rdma_vm_test() {
...
     # Define the pairs using regular arrays
-    pairs=(
-    "0,6"
-    "1,7"
-    "2,8"
-    "3,9"
-    "4,10"
-    "5,11"
-    )
+    elements=($(cat "${PAIRS_FILE}"))
```
Comment (blochl) [inline ngc_rdma_wrapper.sh:231]: Any verification of the content?

File: ngc_rdma_wrapper.sh [hunk 2]
```diff
@@ -240,27 +243,12 @@ ngc_rdma_test_external_loopback() {
...
-            if ! "${scriptdir}/ngc_rdma_test.sh" "${SERVER_IP}" "mlx5_${first}" "${SERVER_IP}" "mlx5_${second}" "${tests}" ${use_cuda} &>> "${LOGFILE}" ; then
-                echo "${RED}Issue with device mlx5_${first} <-> mlx5_${second}" | tee -a "${LOGFILE}${NC}"
-            fi
+    for ((i = 0; i < ${#elements[@]}; i += 2)); do
+        echo -e "${WHITE}=== Device: ${elements[i]} <-> ${elements[i + 1]} ===${NC}" &>> "${LOGFILE}"
+        if ! "${scriptdir}/ngc_rdma_test.sh" "${SERVER_IP}" "${elements[i]}" "${SERVER_IP}" "${elements[i + 1]}" "${tests}" ${use_cuda} &>> "${LOGFILE}" ; then
+            echo -e "${RED}Issue with device ${elements[i]} <-> ${elements[i + 1]}${NC}"  | tee -a "${LOGFILE}"
         fi
-        wrapper_results
+            wrapper_results
```
Comment (blochl) [inline ngc_rdma_wrapper.sh:251]: ?

File: ngc_rdma_wrapper.sh [hunk 3]
```diff
@@ -339,14 +327,19 @@ check_ssh() {
 
 # If 1 host provided, run external loopback test:
 if [[ $# == 1 ]]; then
+    if [[ ! -f "${PAIRS_FILE}" ]]; then
```
Comment (blochl) [inline ngc_rdma_wrapper.sh:330]: This can be done with single `[` (AKA `test`).

File: ngc_rdma_wrapper.sh (hunk 1 above)
Comment (RavidRothfeld) [reply ngc_rdma_wrapper.sh:231]: Added

File: ngc_rdma_wrapper.sh (hunk 2 above)
Comment (RavidRothfeld) [reply ngc_rdma_wrapper.sh]: Fixed

File: ngc_rdma_wrapper.sh (hunk 3 above)
Comment (RavidRothfeld) [reply ngc_rdma_wrapper.sh]: Refactored

**Review by blochl — APPROVED** [review]
Thanks!

**Pull request merged**